bigstream/features.py

```diff
@@ -88,13 +134,19 @@ def get_contexts(image, coords, radius):
 def _stats(arr):
     """
     """
-
-    # compute mean and standard deviation along columns
-    arr = arr.astype(np.float64)
-    means = np.mean(arr, axis=1)
-    sqr_means = np.mean(np.square(arr), axis=1)
-    stddevs = np.sqrt( sqr_means - np.square(means) )
-    return means, stddevs
+    try:
```

Comment:
Do you recall what kind of exceptions you were getting here that you needed to add a try-catch?

Reply:
I don't remember the exact exception - but it was something related to the size of the numpy array